docs/products/m3db/howto/telegraf.rst

```diff
@@ -0,0 +1,51 @@
+How to Write to M3 from Telegraf
+================================
+
+M3 supports multiple methods of writing metrics to the M3 database. Two common options are the InfluxDB wire-protocol, and the Prometheus remote write protocol. It is possible to write metrics from an existing Telegraf setup to M3 using either of these protocols, so we've covered the configuration for both options here.
+
```

Comment:
Should we also add here the **Variables** section we have for the other HowTos and reference the variables in the config files?

Reply:
Oh we can! It's not a code sample, but why not?